SeaPublicWebsite/Controllers/EnergyEfficiencyController.cs

```diff
@@ -454,19 +454,16 @@ public async Task<IActionResult> NoEpcFound_Post(NoEpcFoundViewModel viewModel)
             {
                 return await NoEpcFound_Get(viewModel.Reference);
             }
-            
-            var propertyData = await propertyDataStore.LoadPropertyDataAsync(viewModel.Reference); 
-            
-            var forwardArgs = questionFlowService.ForwardLinkArguments(QuestionFlowPage.ConfirmEpcDetails, propertyData);
-            return RedirectToAction(forwardArgs.Action, forwardArgs.Controller, forwardArgs.Values);
+
+            return await UpdatePropertyAndRedirect(p => { }, viewModel.Reference, QuestionFlowStep.NoEpcFound);
```

Comment:
Note that this used to be `ConfirmEpcDetails` which I think was a mistake (although it went to the same page which I guess is why it wasn't spotted earlier)

Reply:
Ah yep, that's probably on me, definitely a mistake, good spot!

Reply:
Since this does nothing, we can modify the `.cshtml` file so that the button has `href=<nextPageURL>`. That will make the govuk 'button' an `<a>` element, removing the need for an unnecessary post.